### PR TITLE
fix spec tests

### PR DESCRIPTION
**Pull request by rwaffen**

### CODE REVIEW

File: lib/puppet/functions/wildfly/objectify.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Converts a LIST of STRINGs and OBJECTs into an OBJECT of OBJECTs.
+Puppet::Functions.create_function(:'wildfly::objectify') do
+  dispatch :objectify do
+    param 'Array[Variant[String, Hash[String, Any]]]', :input_list
+  end
```
Comment (tuxmea) [inline lib/puppet/functions/wildfly/objectify.rb:5]: Please also add the desired return type. https://www.puppet.com/docs/puppet/7/functions_ruby_signatures#return-type-method

Comment (rwaffen) [reply lib/puppet/functions/wildfly/objectify.rb:5]: okay. was wondering why puppet string says it returns any. so i will add a return type

**Comment by rwaffen** [comment]
okay, don't get the existing acceptance tests to run, so maybe rewrite them from scratch.

image

File: .fixtures.yml [hunk 2]
```diff
@@ -2,9 +2,9 @@ fixtures:
   repositories:
       stdlib:
         repo: "https://github.com/puppetlabs/puppetlabs-stdlib.git"
-        branch: "4.13.1"
```
Comment (bastelfreak) [inline .fixtures.yml:5]: please switch to git branches and not a specific tag. in unit tests we test against unreleased changes, in acceptance tests we use forge releases.

File: .fixtures.yml [hunk 3]
```diff
@@ -2,9 +2,9 @@ fixtures:
...
+        branch: "v9.4.0"
       augeas_core:
         repo: "https://github.com/puppetlabs/puppetlabs-augeas_core.git"
-        branch: "1.0.4"
+        branch: "v1.4.1"
   symlinks:
```
Comment (bastelfreak) [inline .fixtures.yml:9]: the `symlink` isn't required anymore, puppetlabs_spec_helper will create it for us.

**Review by bastelfreak — APPROVED** [review]

**Pull request merged**